### ISSUE TITLE
When creating an animation from a consumable nothing shows.

### ISSUE THREAD

**Issue by ExileofBrokenSky** [otigon/automated-jb2a-animations#126]
I'm trying to set up a template explosion for a consumable item like a grenade. If I apply the same settings to a spell, it works fine, but on a consumable item no animation plays when it is used, though provided sound effects will play. The same no animation will happen if a spell scroll of a working animated spell is used. No errors in the console either.

All systems and modules are updated as of shortly before this post.
![image](https://user-images.githubusercontent.com/28786777/124385894-c05d8100-dca5-11eb-887d-6409449ed354.png)

**Comment by otigon** [comment]
What game system and Foundry version are you on, and have you tested with only A-A active to verify?

**Comment by ExileofBrokenSky** [comment]
A-A must have updated just after i started playing, which was a longish session and i wasn't playing with the grenades till after we finished for the night.
Now that I've updated from 3.something to 4.something, it's got the new interface and the animations from consumables work great now.
Although a different problem presents itself:
I'm using Midi-Qol to automate saving throws in conjunction with Monk's Token Bar, now when I use the grenade, the animation plays fine but the assigned sound doesn't play until after the saves are rolled, and i have it set for the player/myself to have to click the save instead of just rolling it auto asap. So it goes template placement -> animation plays -> Monk's token bar asks for saves -> saves are rolled -> explosion sound effect.
